### DISCUSSION TITLE
Is there support for GitHub Enterprise when fetching number of stars, forks and repo name?

### DISCUSSION THREAD

**Discussion by dellagustin-sap** [squidfunk/mkdocs-material#5402] [Q&A] [answered]
Hello,

MkDocs Material automatically requests and renders stars and forks for public repostories hosted on GitHub and GitLab.
Is there some form of similar support for GitHub Enterprise?

I'm assuming it doesn't, but would you be interested in adding this feature?

Best Regards,
Guilherme.

**Comment by squidfunk** [comment, marked as answer]
Nope, please see this exact duplicate:

- #5157

**Reply by dellagustin-sap** [reply]
Thanks!
